### ISSUE TITLE
GitHub Ruby Style Guide is 404

### ISSUE THREAD

**Issue by apjanke** [Homebrew/brew#106]
The GitHub Ruby Style Guide at https://github.com/styleguide/ruby is returning a 404 error for me today. It's linked from our [`Library/Homebrew/README.md`](https://github.com/Homebrew/brew/blob/144d86f73315f59d65234ace69880bb9d3a90acb/Library/Homebrew/README.md), which is now a dead link.

Anybody know where it's gone, or if this is a transient error? Google still returns this URL as its main hit.

I've reported this to GitHub support through their [support contact form](https://github.com/contact). For now I'm leaving the link as is, in case it's a transient problem, and using the Wayback Machine to read it.

**Comment by apjanke** [comment]
GitHub support responded. They're looking in to it.

**Comment by UniqMartin** [comment]
Any updates from GitHub support? If not, we might want to temporarily (or permanently) change that link to https://github.com/bbatsov/ruby-style-guide which is also the basis for our automated style checking with RuboCop (though we've disabled/relaxed some rules).

**Comment by mikemcquaid** [comment]
It's been taken down temporarily but will be coming up again at some point in the near future, apparently.

**Comment by apjanke** [comment]
Yes, GitHub support replied that the style guide was unshipped, but "[t]his is only temporary, and the styleguides will be back and much improved". (They didn't actually take this long to respond; I got a response a couple days back and it just sat in my inbox due to me being busy and/or disorganized.)

:+1: to changing link to bbatsov's style guide in the mean time, with a note that we used to use GitHub's guide when it existed. GitHub's own style guide was based on it too; the acknowledgment is right up top. (https://web.archive.org/web/20160410033955/https://github.com/styleguide/ruby for reference.)
